### PR TITLE
Add support for OpenSDS authorization token

### PR DESCRIPTION
**Pull request by joseph-v**
**What this PR does / why we need it**:
This PR will add authorization token parameter to volume provision workflow for using with OpenSDS. It also contains minor clean up.

### CODE REVIEW

File: contrib/st2/opensds/actions/list_volume.py [hunk 1]
```diff
@@ -18,7 +18,9 @@
 
 
 class ListVolumeAction(Action):
-    def run(self, ipaddr="", port="", projectid=""):
+    def run(self, ipaddr="", port="", token="", projectid=""):
         url = "http://"+ipaddr+":"+port+"/v1beta/"+projectid+"/block/volumes"
-        r = requests.get(url=url)
+        headers = {'x-auth-token': token}
+        r = requests.get(url=url, headers=headers)
+        print(r.text)
```
Comment (kumarashit) [inline contrib/st2/opensds/actions/list_volume.py]: Are these prints part of results?

Comment (joseph-v) [reply contrib/st2/opensds/actions/list_volume.py]: Yes. List Volume action output to stdout

Comment (kumarashit) [reply contrib/st2/opensds/actions/list_volume.py]: Ok

File: contrib/st2/opensds/actions/attach_volume.py [hunk 2]
```diff
@@ -26,17 +26,20 @@ def run(self,
...
-            tenantid="",
             accessprotocol="",
+            token="",
             volumeid=""):
         data = {
             "Mountpoint": mountpoint,
             "HostInfo": hostinfo,
             "ConnectionInfo": connectioninfo,
-            "TenantId": tenantid,
+            "TenantId": projectid,
```
Comment (wisererik) [inline contrib/st2/opensds/actions/attach_volume.py]: we use tenantid not projectid

Comment (joseph-v) [reply contrib/st2/opensds/actions/attach_volume.py]: Done

**Pull request merged**